src/Components/CommentInfo.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Comment } from '../types/Comment';
+
+export const CommentInfo: React.FC<Comment> = ({ name, email, body }) => (
```

Comment:
```suggestion
export const CommentInfo: React.FC<Props> = ({ name, email, body }) => (
```
Пропиши type Props і використовуй саме його для пропсів